guests/invitation.py

```diff
@@ -49,7 +49,7 @@ def send_invitation_email(party, test_only=False, recipients=None):
     context['couple'] = settings.BRIDE_AND_GROOM
     template_html = render_to_string(INVITATION_TEMPLATE, context=context)
     template_text = "You're invited to {}'s wedding. To view this invitation, visit {} in any browser.".format(
-        settings.BRIDE_AND_GROOM,
+        settings.BRIDE_AND_GROOM, settings.WEDDING_WEBSITE_URL +
```

Comment:
I probably would have put this on the same line as the `reverse` for clarity. Not a big deal, but I'll give you a chance to respond if you agree it's better.

Reply:
I totally blame it on autoformat and not on laziness :D.